tests/plots/test_image.py

```diff
@@ -43,6 +43,61 @@ def test_image_multi(imagenet50_example):
     return plt.gcf()
 
 
+@pytest.mark.mpl_image_compare
+def test_image_multi_tf_bug():
+    # GH 3874
+    tf = pytest.importorskip("tensorflow")
+    ssl = pytest.importorskip("ssl")
+
+    ssl._create_default_https_context = ssl._create_unverified_context
+    (x_train, y_train), (x_test, y_test) = tf.keras.datasets.cifar10.load_data()
+    # reshape and normalize data
+    subset_size = 200
+    x_train_subset = x_train[:subset_size]
+    y_train_subset = y_train[:subset_size]
+    x_test_subset = x_test[:subset_size]
+    y_test_subset = y_test[:subset_size]
+
+    # Normalize the data
+    x_train_subset = x_train_subset.astype("float32") / 255
+    x_test_subset = x_test_subset.astype("float32") / 255
+
+    # Reshape the labels
+    y_train_subset = y_train_subset.reshape(
+        subset_size,
+    )
+    y_test_subset = y_test_subset.reshape(
+        subset_size,
+    )
+
+    # define the model architecture
+    inputs = tf.keras.Input(shape=(32, 32, 3))
```

Comment:
we have a problem here. Retraining this won't reproduce the image. Any ideas how we can get this to work? I would like to reproduce the issue from the ticket